examples/v1beta1/pipelineruns/pipelinerun-task-execution-status.yaml

```diff
@@ -42,3 +42,17 @@ spec:
               if [[ $(params.task1Status) != "Succeeded" ||  $(params.task2Status) != "None" ]]; then
                 exit 1;
               fi
+    - name: task4 # this task verifies the aggregate status of all tasks, it fails if verification fails
+      params:
+        - name: aggregateStatus
+          value: "$(tasks.status)"
+      taskSpec:
+        params:
+          - name: aggregateStatus
+        steps:
+          - image: alpine
+            name: verify-aggregate-tasks-status
+            script: |
+              if [[ $(params.aggregateStatus) != "Completed" ]]; then
+                exit 1;
+              fi
```

Comment:
please add an example demonstrating using aggregate status in whenexpressions as well

Reply:
sure, added an example in [pipelinerun-with-when-expressions.yaml](https://github.com/tektoncd/pipeline/pull/3817/files#diff-7627b6a35e7abe827131f78979e678fc098c67410c55c1b072a2bf8ddf354bd4) ⬇️